whois-commons/src/main/resources/nrtm_schema.sql

```diff
@@ -12,41 +12,68 @@
 DROP TABLE IF EXISTS `source`;
 create table `source`
 (
```

Comment:
We usually put all our sql files including patch files in whois-commons resources.

Reply:
yes, that's where it is already :) or do you mean actually in the 'patch' directory?